text/3-entrees/basic-pow.md

```diff
@@ -0,0 +1,230 @@
+# Basic Proof of Work
+*[`nodes/basic-pow`](https://github.com/substrate-developer-hub/recipes/tree/master/nodes/basic-pow)*
+
+The `basic-pow` node uses a minimal [Proof of Work](https://en.wikipedia.org/wiki/Proof_of_work) consensus engine to reach agreement over the blockchain. This node is kept intentionally simple. It omits some features that make Proof of Work practical for real-world use such as difficulty adjustment and block rewards. Nonetheless, it is a real usable consensus engine that will teach us many useful aspects of dealing with consensus and prepare us to understand more advanced consensus engines in the future. In particular we will learn about:
+* Substrate's [`BlockImport` trait](https://substrate.dev/rustdocs/master/sp_consensus/block_import/trait.BlockImport.html)
+* Substrate's [import pipeline](https://substrate.dev/rustdocs/master/sp_consensus/import_queue/index.html)
+* Structure of a typical [Substrate Service](https://substrate.dev/rustdocs/master/sc_service/index.html)
+* Configuring [`InherentDataProvider`](https://substrate.dev/rustdocs/master/sp_authorship/struct.InherentDataProvider.html)s
+
+## The Structure of a Node
+
+You may remember from the [hello-substrate recipe](../2-appetizers/1-hello-substrate.md) that a Substrate node has two parts. An outer part that is responsible for gossiping transactions and blocks, handling [rpc requests](./custom-rpc.md), and reaching consensus. And a runtime that is responsible for the business logic of the chain. This architecture diagram illustrates the distinction.
+
+![Substrate Architecture Diagram](../img/substrate-architecture.png)
+
+In principle the consensus engine, part of the outer node, is agnostic over the runtime that is used with it. But in practice, most consensus engines will require the runtime to provide certain [runtime APIs](./runtime-api.md) that affect the engine. For example Aura, and Babe, query the runtime for the set of validators. A more real-world PoW consensus would query the runtime for the bock difficulty. Additionally, some runtimes, rely on the consensus engine to provide [PreRuntime Digests](https://substrate.dev/rustdocs/master/sp_runtime/generic/enum.DigestItem.html#variant.PreRuntime). For example Runtimes that include the Babe pallet, expect a preruntime digest containing information about the current babe slot. Because of these requirements, this node will use a dedicated `pow-runtime`. The contents of that runtime should be familiar, and will not be discussed here.
```

Comment:
```suggestion
In principle the consensus engine, part of the outer node, is agnostic over the runtime that is used with it. But in practice, most consensus engines will require the runtime to provide certain [runtime APIs](./runtime-api.md) that affect the engine. For example, Aura and Babe query the runtime for the set of validators. A more real-world PoW consensus would query the runtime for the bock difficulty. Additionally, some runtimes, rely on the consensus engine to provide [pre-runtime digests](https://substrate.dev/rustdocs/master/sp_runtime/generic/enum.DigestItem.html#variant.PreRuntime). For example, runtimes that include the Babe pallet expect a pre-runtime digest containing information about the current babe slot. Because of these requirements, this node will use a dedicated `pow-runtime`. The contents of that runtime should be familiar, and will not be discussed here.
```